EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -135,63 +142,146 @@ public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch
             //mlContext.Model.Save(trainedModel, trainingData.Schema, @"C:\Users\User\Downloads\TrainedModel.zip");
         }
 
-        private static Dictionary<string, int> GetSequenceToPSMCount(List<PeptideSpectralMatch> psms)
+        private static Dictionary<string, Dictionary<int, Tuple<double, double>>> ComputeHydrophobicityValues(List<PeptideSpectralMatch> psms, bool modified)
```

Comment:
change `modified` to something more clear like `computingHydrophobicitiesForModifiedPeptides`, it confused me on line 163